### PR TITLE
Changes to be able to use 2.2 on ESRF ID30A3

### PR DESCRIPTION
**Pull request by mguijarr**
I am preparing the transition to 2.2 at ESRF (finally !), so here are the latest changes from ID30A3
I would like to get in 2.2.

### CODE REVIEW

File: QueueModel.py [hunk 1]
```diff
@@ -48,6 +48,13 @@ def __getstate__(self):
     def __setstate__(self, d):
         self.__dict__.update(d)
 
+    def __getstate__(self):
+        d = dict(self.__dict__)
+        return d
+
+    def __setstate__(self, d):
```
Comment (IvarsKarpics) [inline QueueModel.py:55]: Looks like **setstate** and **getstate** is defined several times. Could you please clear this so I can accept the PR?

Comment (mguijarr) [reply QueueModel.py:55]: Hi Ivars :)
Look at the final code, not the diff : I think it's cleared in fact ?

Comment (IvarsKarpics) [reply QueueModel.py:55]: Hmm, not sure. Also in the final code is defined 3 times. Ok, I will accept the PR and then check the final version

Comment (mguijarr) [reply QueueModel.py:55]: You're right - I will fix it today hopefully ; thanks for having accepted the PR see you :D

Comment (mguijarr) [reply QueueModel.py:55]: See #126

**Pull request merged**